keps/sig-release/1498-kubernetes-yearly-support-period/kep.yaml

```
@@ -22,10 +22,10 @@ approvers:
   - "@deads2k"
   - "@dchen1107"
   - "@pwittrock"
-editor: "@youngnick"
+editor: "@tpepper"
 creation-date: 2020-01-22
-last-updated: 2020-01-22
-status: provisional
+last-updated: 2020-05-15
```

```suggestion
latest-milestone: v1.19
```

`last-updated` field is deprecated.

---

updated